packages/zip-it-and-ship-it/tests/main.test.ts

```diff
@@ -896,18 +896,6 @@ describe('zip-it-and-ship-it', () => {
     },
   )
 
-  testMany(
-    'Exposes the main export of `node-fetch` when imported using `require()`',
-    [...allBundleConfigs],
```

Comment:
Whew, it took me a few hops to find this but here's where this was introduced: https://github.com/netlify/zip-it-and-ship-it/pull/353. I'm a little worried that I don't fully understand the intent of this, so maybe we should wait for a thumbs up from @eduardoboucas on this one.